src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
+     */
+    void kick();
+
+    /**
+     * Kicks the player given a reason.
+     *
+     * @param reason The reason for the kick
+     */
+    void kick(String reason);
```

Comment:
@gratimax IMO The point is not what is stored somewhere or somehow, because that can change very soon.

We have `Message.Text` or `Text.Literal` for Colored Strings, which supports all required formatting for the given message. I just find it very strange to have only one place in the entire APi that supports colored messages, but is not using Message for that. Using `kick(coloredMessage.toLiteral())` or `kick("§4Do not Spam")` looks weird.

But its probably easier to use it from commands that way... well is there somewhere a `TextStyle.getLegacyChar()` method to do the replacement from `&` to `§` myself?